pro-planner-client/public/index.html

```diff
@@ -1,21 +1,25 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <meta name="theme-color" content="#000000" />
-    <meta
-      name="description"
-      content="Web site created using create-react-app"
-    />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
-    <!--
+	<head>
+		<meta charset="utf-8" />
+		<link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+		<meta name="viewport" content="width=device-width, initial-scale=1" />
+		<meta name="theme-color" content="#000000" />
+		<meta
+			name="description"
+			content="Web site created using create-react-app"
+		/>
+		<link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+		<script
+			src="https://kit.fontawesome.com/08256dca2b.js"
```

Comment:
I'm not too familiar with what this is, are we pulling for a font/icons here? Also, in this project what are we using this for?

Reply:
Not 100% sure about this. @heeseop-an Do you know what this may be for? 
I installed something called react-icons which I imported in LandingPage.jsx, however i dont think thats related to this.

Reply:
I'm not sure either. I don't think we are using that script.

Reply:
I will try commenting and seeing if it breaks anything. If so, I'll remove.